packages/renderer/src/lib/preferences/PreferencesConnectionCreationOrEditRendering.svelte

```diff
@@ -108,6 +111,23 @@ $effect(() => {
   }
 });
 
+// Re-audit connection when provider status changes
+$effect(() => {
+  // Access the providerInfos store to track its changes
+  $providerInfos;
```

Comment:
question: This statement makes the effect run when `providerInfos` changes?

Reply:
checked, it is not necessary to call this statement